server/app/routers/api/users/router.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+
+const router = express.Router();
+
+/* ************************************************************************* */
+// Define Your API Routes Here
+/* ************************************************************************* */
+
+// Import offer-related actions
+const { browse, signupUser } = require("../../../controllers/UserAction");
+
+// Route to get a list of offers
+router.get("/", browse);
+
+router.post("/signup", signupUser);
```

Comment:
We need to check what the norm for routes and methods' names.
I would have put **router.post("/", add)**

Reply:
done